index.js

```diff
@@ -58,7 +58,17 @@ var handlers = {
         this.emit('AMAZON.StopIntent');
     },
     'AMAZON.HelpIntent': function () {
-        this.emit('Unhandled');
+        var self = this;
+        ApiAi.eventRequest({name: 'HELP'}, {sessionId: alexaSessionId})
+            .on('response', function (response) {
+                var speech = response.result.fulfillment.speech;
+                self.emit(':ask', speech, speech);
```

Comment:
I was already handling `AMAZON.HelpIntent` by calling `Unhandled` which triggers the [`FALLBACK` event](https://github.com/Gnzlt/AlexaApiAiBridge/blob/master/index.js#L77) in Api.ai. 
I think it's the same idea that you want to implement with `HELP` intent.